.github/workflows/doc-build.yml

```diff
@@ -227,7 +227,7 @@ jobs:
         shell: bash
         run: |
           export PYTHONFAULTHANDLER=1
-          xvfb-run make -C doc ${{ inputs.builder }} SPHINXOPTS="-j auto -W --keep-going" | tee doc_build.log
+          xvfb-run make -C doc ${{ inputs.builder }} SPHINXOPTS="-j auto" | tee doc_build.log
```

Comment:
Shall we get back `-W --keep-going` ??

Reply:
There are still warnings in the documentation build process. At the moment, we cannot get ``-W --keep-going`` back.

Reply:
Ok ok.